pages/query.js

```diff
@@ -128,6 +130,35 @@ export default function Query() {
       <Navigation />
       <Breadcrumbs />
       <PagePreamble />
+      <label className="m-2 relative inline-flex cursor-pointer select-none items-center">
+        <input
+          type="checkbox"
+          checked={!isGrch38}
+          onChange={(e) => {
+            setIsGrch38(!e.target.checked);
+          }}
+          className="sr-only"
+        />
+        <span
+          className={`${grch38LabelStyle}  flex items-center text-xl font-bold`}
+        >
+          GRCh38
+        </span>
+        <span
+          className={`mx-4 flex h-8 w-[60px] items-center rounded-full p-1 duration-200 ${"bg-[#CCCCCE]"}`}
```

Comment:
I think `${"bg-[#CCCCCE]"}` is the same thing as just `bg-[#CCCCCE]`. It looks like this whole className string can just be a static string.

Reply:
will update it. Thanks!